src/main/extras/app/entrypoint.sh

```diff
@@ -3,6 +3,7 @@
 set -x
 set -e
 
+USRFILE="/tmp/jmxremote.access"
 PWFILE="/tmp/jmxremote.password"
 function createJmxPassword() {
```

Comment:
I think this function should be renamed now that its responsibilities have been expanded, what do you think?

Reply:
yes, definitely makes sense!